nttt/tidyup.py

```diff
@@ -1,5 +1,5 @@
 from .utilities import find_files, find_replace, find_snippet, get_file, save_file
-
+from .utilities import trim_spaces_on_specific_markdown
```

Comment:
I would suggest to put this code in a different source file. Something like `cleanup_markup.py`. 

Let's keep only general infrastructure methods for reading, saving, searching etc in `utitlites.py`. And nttt-specific business logic elsewhere.